e2e/fixtures/rsc-css-modules/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "rsc-css-modules",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "waku dev",
+    "build": "waku build",
+    "start": "waku start"
+  },
+  "dependencies": {
+    "react": "19.0.0-beta-4508873393-20240430",
+    "react-dom": "19.0.0-beta-4508873393-20240430",
+    "react-server-dom-webpack": "19.0.0-beta-4508873393-20240430",
+    "waku": "workspace:*"
+  },
+  "devDependencies": {
+    "@types/react": "18.2.74",
+    "@types/react-dom": "18.2.24",
```

```suggestion
    "@types/react": "18.3.1",
    "@types/react-dom": "18.3.0",
```
but maybe it's not related?

---

I updated it but chose to not push the commit until the test was 100% passing locally so I do not waste your CI